### PR TITLE
DO-2471 - Fix jaas auth module

### PR DESCRIPTION
**Pull request by ls-avanier**
As title implies.

### CODE REVIEW

**Comment by ls-brentsmith** [comment]
Could you explain what that change is doing?

From what-auth to what-auth?

File: templates/default/jaas-loginmodule.conf.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RDpropertyfilelogin {
+org.eclipse.jetty.plus.jaas.spi.PropertyFileLoginModule required
+debug="true"
```
Comment (ls-brentsmith) [inline templates/default/jaas-loginmodule.conf.erb]: also, is debug necessary or a remnant of figuring it out?

**Comment by ls-avanier** [comment]
From `org.eclipse.jetty.jaas.spi.PropertyFileLoginModule required` to `org.eclipse.jetty.plus.jaas.spi.PropertyFileLoginModule required`.

`debug="true"` is the default value :( ... but I could remove it! :smiling_imp:

**Comment by ls-brentsmith** [comment]
👍

**Comment by ls-avanier** [comment]
The default module is not present in our Jetty version.
For reference, the info can be found [here](http://rundeck.org/docs/upgrading/index.html#jetty-embedded-server-was-upgraded-to-9.0.x)

**Pull request merged**